src/utils/daos.c

```diff
@@ -552,6 +553,7 @@ common_op_parse_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 		{"user",	required_argument,	NULL,	'u'},
 		{"group",	required_argument,	NULL,	'g'},
 		{"principal",	required_argument,	NULL,	'P'},
+		{"include-acl",	no_argument,		NULL,	DAOS_INCLUDE_ACL_OPTION},
```

Comment:
(style)  line over 80 characters

Reply:
This warning is ignored for another case in this command table -- keeping the table readable may trump keeping lines under 80 characters.